apps/vaporgui/VIntSliderEdit.h

```diff
@@ -28,6 +31,9 @@ class VIntSliderEdit : public VSliderEditInterface {
 
     void AllowUserRange(bool allowed = true);
 
+    //! Allow the toggling of dynamic updating widgets
+    void AllowDynamicUpdate() const;
```

Comment:
The V-widget does not have a concept of params so it probably shouldn't have a function pertaining to the update frequency of params?

Reply:
I understand, but the V-Widget always outputs Changed signals and IntermediateChange signals regardless, so how does the checkbox affect it?

Reply:
@stasj - Sorry, I'm unclear on this.  Are you saying you'd like IntermediateChange signals to be disabled unless AllowDynamicUpdate() is called?  Something like:

`void VIntSliderEdit::AllowDynamicUpdate() const { 
    _menu->AllowDynamicUpdate();
    disconnect(_slider, &VSlider::ValueChangedIntermediate, this, &VIntSliderEdit::_sliderChangedIntermediate);
}`

Reply:
AllowDynamicUpdate pertains to Dynamically Updating the Params. The V-Widgets are supposed to be unrelated to the Params and therefore should not have functions pertaining to updating the Params.

Reply:
Please read my note above, regarding refactoring.  I'll schedule a meeting on Monday so we can discuss.